tests/factory_fixtures/property.py

```diff
@@ -2,18 +2,28 @@
 from models.property import PropertyModel
 
 @pytest.fixture
-def create_property(create_property_manager):
-    def _create_property(pm=create_property_manager()):
-        property = PropertyModel(
-                name='the heights',
-                address='111 SW Harrison',
-                city="Portland",
-                unit="101",
-                state="OR",
-                zipcode="97207",
-                propertyManagerIDs=[pm.id],
-                archived=False
-            )
+def property_attributes(faker):
+    def _property_attributes(archived=False, pm=None):
+        return {
+            'name': faker.unique.name(),
+            'address': faker.address(),
+            'city': faker.city(),
+            'unit': faker.building_number(),
+            'state': faker.state() ,
+            'zipcode': faker.postcode(),
+            'propertyManagerIDs': [pm.id] if pm and pm.id else [],
+            'archived': archived,
+        }
+    yield _property_attributes
+
+@pytest.fixture
+def create_property(property_attributes, create_property_manager):
+    def _create_property():
```

Comment:
This looks excellent. 
Probably keep in mind that we may want to update this to allow the developer to specify specific property managers when creating a property, but that road can be crossed if the need arises.